kratos/solving_strategies/convergencecriterias/displacement_criteria.h

```diff
@@ -82,15 +83,38 @@ class DisplacementCriteria : public ConvergenceCriteria< TSparseSpace, TDenseSpa
     ///@name Life Cycle
     ///@{
 
+    /** Constructor.
+    */
+    explicit DisplacementCriteria(Kratos::Parameters Settings)
+        : BaseType()
+    {
+        if (Settings.Has("displacement_absolute_tolerance")) {
+            mAlwaysConvergedNorm = Settings["displacement_absolute_tolerance"].GetDouble();
+        } else if (Settings.Has("absolute_tolerance")) {
```

Comment:
Why either or?

Reply:
What_?

Reply:
The names are so confusing that I not sure about anything right now

Reply:
Then I think you should stick to our usual convention

anyway this is not the scope of this PR

Reply:
I am quite sure that it follows our convention

Reply:
the double-definition does not follow our convention => this is the thing I think we should avoid

So again my question, why doing this? :)
I guess in order to be independent of which criteria you create?

Reply:
Well, a priori you don't know the criterias you are combining